### ISSUE TITLE
Cryogenic storage requires some restrictions

### ISSUE THREAD

**Issue by marboww** [space-wizards/space-station-14#29328]
## Description

Cryogenic storage can bring in not only people, but also other entities. This is dangerous because you can safely put Ian in storage and delete the high-risk item forever. I propose to ban the introduction of entities other than people.

**Comment by Brandon-Huu** [comment]
Ban the people who put ian into storage

**Comment by Brandon-Huu** [comment]
also, ian is not an item

**Comment by lzk228** [comment]
> Ban the people who put ian into storage

yes i agree, but most maintainers want the game be more "admin-free", and there is even an issue about converting rules to mechanics #20064

**Comment by slarticodefast** [comment]
Resolved by https://github.com/space-wizards/space-station-14/pull/30574
